.github/workflows/nightly-remote-benchmarks .yml

```diff
@@ -0,0 +1,15 @@
+# Run benchmarks on a remote system for the nightly (edge) build
+# - Calls the reusable worflow remote-benchmarks.yml
+
+name: Nightly Benchmark Test on Docker Deephaven
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
+
+jobs:
+  process-release-benchmarks:
+    uses: ./.github/workflows/remote-benchmarks.yml
```

Comment:
I've not used this pattern before, cool!

Reply:
I think this is relatively new.  One caveat, unlike most other actions, relative paths must have the "./" as a prefix.  Doesn't seem to work without it.